src/main/java/org/apache/accumulo/access/AeNode.java

```diff
@@ -249,7 +252,7 @@ private OrNode(List<AeNode> children) {
 
     @Override
     char operator() {
-      return '|';
+      return OR_OPERATOR;
```

Comment:
This method returns a `char`, so I think we want to leave it unless from where it's called its being cast to a `byte`.